Monika After Story/game/pong.rpy

```diff
@@ -564,10 +564,10 @@ label demo_minigame_pong:
         m "Would you like to play again?{fast}"
 
         "Yes.":
-            $ pong_ev = mas_getEV("mas_pong")
-            if pong_ev:
-                # each game counts as a game played
-                $ pong_ev.shown_count += 1
+            $ mas_setEVPropValues(
+                "mas_pong",
+                mas_getEVPropValue("mas_pong", "shown_count", 0) + 1
```

Comment:
Similar to the long comment I added, I wonder how much better this really is. We went from a clear assign-increment operation `+= 1` to a get, add 1, then set operation. Maybe might be more readable to create another EVPropValue function specific to increments/decrements, but then that would need try except or something to handle non-integer cases. That honestly seems like more work for very little gain. I'm fine keeping this as it is, but I'm really not enthusiastic about using this or telling others to use this in assign-modify operations.

Also, `mas_setEVPropValues` expects keyword args, so this should be like:
 ```suggestion
                 shown_count=mas_getEVPropValue("mas_pong", "shown_count", 0) + 1
```